src/pages/Widget/Preview.tsx

```diff
@@ -10,10 +12,13 @@ export default function Preview({ classes = "" }) {
   const endowName = endowment.name || "this nonprofit";
 
   return (
-    <section className={`${classes} @container/preview pb-4`}>
-      <h2 className="text-lg @4xl/widget:text-2xl text-center @4xl/widget:text-left mb-3">
-        That's what our Donation Form looks like:
-      </h2>
+    <section className={`${classes} @container/preview pb-4 mt-10`}>
```

Comment:
```suggestion
    <section className={`${classes} @container/preview pb-4`}>
```
classes affecting outside space should be outside e.g.

```jsx
 <div className="w-full">
      <Configurer endowment={endowment} />
      <Snippet classes="mt-10" />
 </div>
 <Preview classes="mt-10 .... existing classes" />
```